ci/ansible/roles/jenkins-user-add/README.md

```diff
@@ -0,0 +1,21 @@
+jenkins-user-add
+=========
+
+Add the jenkins ci-user on a targeted system.
+
+When executed, this role will do the following:
+
+1. Create the `jenkins` group.
+2. Create the `jenkins` user.
+3. Create the `jenkins` .ssh/ directory.
+4. Add `authorized_keys` from the ansible_user's `authorized_keys`
+5. Verify sub-sudoers directories can be created in /etc/sudoers.
+6. Add `jenkins` to the /etc/sudoers.d/jenkins file
+
+Sample usage within a playbook:
+
+```yaml
```

Comment:
@kersommoura is the `yaml` supposed to be here? I ripped it from `pulp-fips`, but it looks weird.

Reply:
@bherrin3 I did not get the question.